extensions/soundboard/src/utils.ts

```diff
@@ -10,7 +10,7 @@ export const playSoundFromIndex = async (index: number) => {
     return;
   }
 
-  await showHUD(`❌ Favourite #${index} is not defined`);
+  launchCommand({ name: "index", type: LaunchType.UserInitiated, context: { index: index } });
```

Comment:
What do we use the `index` context for?

Reply:
Nothing (at the moment) but if I don't pass anything as context it's triggers the toast since this extension always is UserInitiated, but it could be anything actually

Reply:
Oh, I see! This is a bit of hack to know whether the view was opened from a "user" or from a "launchCommand"

/cc @felixb1010 would be nice if there was a more intuitive way to do this.